frontend/discourse/app/components/bread-crumbs.gjs

```diff
@@ -205,7 +205,7 @@ export default class BreadCrumbs extends Component {
       @name="bread-crumbs-right"
       @connectorTagName="li"
       @outletArgs={{lazyHash
-        tagId=this.tag.id
+        tagId=this.tag.name
```

Comment:
I think we should deprecate the `tagId` outlet argument here in favor of `tagName`. We can do so via the `@deprecatedArgs` argument.

https://github.com/discourse/discourse/blob/9abdc35ef82f4deb1b63ceb5f004d4570375941a/frontend/discourse/app/components/header/contents.gjs#L115-L125